public_html/i18n/en.json

```diff
@@ -0,0 +1,70 @@
+  "record-compare": "Compare",
+  "record-ithenticate": "iThenticate report",
+  "record-url-text": "$1 of edit ($2 words)",
+  "record-noeditor": "No editor found",
+  "record-noeditor-tooltip": "The revision may have been deleted or the data is not yet available in Labs database.",
+  "record-ores": "<a href=\"https://meta.wikimedia.org/wiki/Objective_Revision_Evaluation_Service\" target=\"_blank\">ORES</a> score: $1",
+  "compare-article": "Article",
+  "compare-article-loading": "Loading article revision...",
+  "compare-source": "Source",
+  "compare-source-loading": "Loading possible copyvio source...",
+  "footer-loadmore": "Load More",
+  "footer-nomore": "No more records!",
+  "footer-broughtby": "Brought to you by <a href=\"https://meta.wikimedia.org/wiki/Community_Tech\">Community Tech</a>",
+  "footer-poweredby": "Powered by <a href=\"https://en.wikipedia.org/wiki/User:EranBot\">EranBot</a> and <a href=\"https://en.wikipedia.org/wiki/Wikipedia:Turnitin\">Turnitin</a>",
+  "footer-source": "<a href=\"https://github.com/wikimedia/PlagiabotWeb\">Source</a>",
+  "footer-phabricator": "<a href=\"https://phabricator.wikimedia.org/project/profile/1638\">Phabricator</a>",
+  "footer-imageattr": "<a href=\"#\" data-toggle=\"modal\" data-target=\"#image-attribution\">Image attribution</a>",
+  "lboard-title": "Leaderboard",
+  "lboard-week": "Cases closed in the<br/> last 7 days",
+  "lboard-month": "Cases closed in the<br/> last 30 days",
+  "lboard-alltime": "All time<br/> cases closed",
+  "lboard-back": "Back to CopyPatrol",
+  "imageattr-title": "Image attribution",
+  "imageattr-close": "Close",
+  "imageattr-logo": "<a target=\"_blank\" href=\"https://commons.wikimedia.org/wiki/File:Dialog-error-copyright.svg\">CopyPatrol logo</a> available to the <a target=\"_blank\" href=\"https://en.wikipedia.org/wiki/Public_domain\">public domain</a> courtesy of <a target=\"_blank\" href=\"https://commons.wikimedia.org/wiki/User:Xander89\">Xander89</a> and the <a target=\"_blank\" href=\"http://tango.freedesktop.org/Tango_Desktop_Project\">Tango DesktopProject</a>.",
+  "imageattr-font": "<a target=\"_blank\" href=\"https://fonts.google.com/specimen/Bitter?query=bitter\">Bitter font</a> released under the <a target=\"_blank\" href=\"http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&amp;id=OFL_web\">OpenFont License </a>courtesy of <a target=\"_blank\" href=\"http://huertatipografica.com/\">Huerta Tipográfica</a>.",
+  "alert-testing": "This is a staging environment. Please do not use this tool to record your reviews. For the actual CopyPatrol tool, please use <a href=\"https://tools.wmflabs.org/copypatrol/\">tools.wmflabs.org/copypatrol</a>.",
+  "js-unauthorized": "You need to be logged in to be able to review.",
+  "js-dberror": "There was an error in connecting to database.",
+  "js-undo-own-only": "You can only undo your own reviews.",
+  "js-unknown-error": "An unknown error occurred when loading results. Please try again."
```

Comment:
The only thing is I would avoid including links and markup in the messages. E.g. you get rid of `footer-source` and instead just have `"source": "Source"` and our HTML, `<a href=\"https://github.com/wikimedia/PlagiabotWeb\">{{ 'source'|message }}</a>`. The markup is not translatable and there's a chance someone might unintentionally break it.

And for `footer-poweredby` you could use: `Powered by $1 and $2` since EranBot and Turnitin are also not translatable. If they are, make a separate message for `turnitin` then in our HTML:
`{{ 'footer-poweredby'|message | ( <a href="https://en.wikipedia.org/wiki/Wikipedia:Turnitin">{{ 'turnitin'|message }}</a> | <a href="https://en.wikipedia.org/wiki/User:EranBot">{{ 'EranBot'|message }}</a> }}` (this is pseudocode, not sure how SimpleI18n works)